delphinus/.env.example

```diff
@@ -25,6 +25,7 @@ ENV_APACHE_CONF_DIR=/path/to/conf/files
 # SSL
 ENV_SSL_CERT_DIR=~/certs
 ENV_APACHE_CONTAINER=kingstonduo/apache:apache-shib
+ENV_APACHE_VIRT_CONF_FILE=virt-apache-shib.conf
```

Comment:
Is this still needed?

Reply:
Nope, it is not.  If you can trash this one-line change here, feel free - otherwise I'll take care of it.